salt/utils/yamldumper.py

```diff
@@ -27,7 +28,7 @@
     HAS_IOFLO = False
 
 
-class IndentMixin(object):
+class IndentMixin(SafeDumper):
```

Comment:
For generality, it might be nicer for the mixin to inherit from just `Dumper` instead of `SafeDumper` (I tested this locally, and it works for me). I'm not sure if this will still pass the docs build, however.